packages/testlab/src/http-server-config.ts

```diff
@@ -6,14 +6,20 @@
 /**
  * Create an HTTP-server configuration that works well in test environments.
  *  - Ask the operating system to assign a free (ephemeral) port.
- *  - Use IPv4 localhost `127.0.0.1` on Travis-CI to avoid known IPv6 issues.
+ *  - Use IPv4 localhost `127.0.0.1` to avoid known IPv6 issues in Docker-based
+ *    environments like Travis-CI.
  *
  * @param customConfig Additional configuration options to apply.
  */
 export function givenHttpServerConfig<T extends object>(
   customConfig?: T,
-): T & {host?: string; port: number} {
-  const defaults = {port: 0};
-  const hostConfig = process.env.TRAVIS ? {host: '127.0.0.1'} : {};
-  return Object.assign(defaults, hostConfig, customConfig);
+): T & {host: string; port: number} {
+  const defaults = {
+    host: '127.0.0.1',
```

Comment:
This is problematic as it pretty much excludes our test coverage on ipv6 unless we explicitly add more tests for both ipv4 and ipv6.